Flat/src/GeneralTree.cc

```diff
@@ -88,6 +89,7 @@ void GeneralTree::SetAuxTree(TTree *t) {
   for (auto p : ecfParams) { 
     TString ecfn(makeECFString(p));
     t->Branch("fj1"+ecfn,&(fj1ECFNs[p]),"fj1"+ecfn+"/F");
+    t->Branch("fj1Gen"+ecfn,&(fj1ECFNs[p]),"fj1Gen"+ecfn+"/F");
```

Comment:
We need an additional flag for creating and booking these. The ECFs are about half the tree right now, so doubling them is significant. I want it to be possible to run fat jets without gen fat jets (and you want this too - the reclustering is slow)

Reply:
Actually why do you even want these added to the auxiliary tree? What are you doing with that?